src/batched/KokkosBatched_Gemm_Serial_Impl.hpp

```diff
@@ -159,12 +159,13 @@ namespace KokkosBatched {
       const int
         m = C.dimension(0),
         n = C.dimension(1),
-        k = A.dimension(1);
+        k = A.dimension(0),
 
       static_assert(is_vector<vector_type>::value, "value type is not vector type");      
       static_assert(vector_type::vector_length == 4 || vector_type::vector_length == 8,
                     "AVX, AVX2 and AVX512 is supported");
       const MKL_COMPACT_PACK format = vector_type::vector_length == 8 ?  MKL_COMPACT_AVX512 : MKL_COMPACT_AVX;
+>>>>>>> upstream/develop
```

Comment:
delete this stuffs.

Reply:
Ah missed htose last 2, my mistake.

Reply:
Hold on, I found some more issues.

Reply:
Obviously if the 'make install-lib' and 'make test' commands would have alerted me to these compiler errors, I would have fixed them before committing. I am not sure why it hasn't been detecting them and that I've had to eyeball them instead.

Reply:
this code is instanciated when you enable mkl as its third party library. that is why you cannot catch during compiling.

Reply:
Ok. Strange, I didn't know that. That has been the root of all this trouble.

Reply:
Ah I see the MKL preprocessor ifdefs surrounding that method, makes sense to me.